mitodl/open-discussions
# Fixed bug with post summary showing raw markdown

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1739

#### What's this PR do?
See title

#### How should this be manually tested?
- Check that a new text post w/ markdown has a text preview on a post listing with no markdown
- Recreate your index and check and existing text post w/ markdown for the same thing

#### Screenshots (if appropriate)
![ss 2019-01-25 at 12 03 04](https://user-images.githubusercontent.com/14932219/51762196-107b9680-209d-11e9-84d5-f9dabc97b9f3.png)
![ss 2019-01-25 at 12 02 53](https://user-images.githubusercontent.com/14932219/51762197-107b9680-209d-11e9-81aa-6052003b6952.png)
